server/src/inmantals/server.py

```diff
@@ -229,6 +251,60 @@ async def textDocument_references(self, textDocument, position, context):  # noq
 
         return [self.convert_location(loc.data) for loc in range]
 
+    async def workspace_symbol(self, query: str) -> List[Dict[str, object]]:
+        if self.types is None:
+            return []
+
+        query_upper: str = query.upper()
```

Comment:
I tried to mimic pucharm's behavior (because that's the only reference I've got and it feels useful). Pycharm matches case-insensitively and partially. Would you prefer a different behavior?

Reply:
I agree. Matching partially and case-insensitive makes sense.